blocks/api/registration.js

```diff
@@ -118,15 +118,20 @@ export function registerBlockType( name, settings ) {
 	if ( ! settings.icon ) {
 		settings.icon = 'block-default';
 	}
-	settings = {
-		name,
-		attributes: get( window._wpBlocksAttributes, name ),
+
+	const attributes = settings.attributes ?
```

Comment:
Was the change to extract this out necessary? Is it just for providing the default to `get`, and if so, could that have been added in-place?

Reply:
No reason, just a rebase leftover